app/communities-configs/filecoin/index.js

```diff
@@ -0,0 +1,318 @@
+import React from 'react';
+import { css } from 'styled-components';
+
+import FileiconLogo from './images/filecoin-logo.svg?inline';
+import favicon from '!file-loader?name=[name].[ext]!images/favicon-filecoin.ico';
+
+import RobotoRegularEOT from './fonts/Roboto-Regular.eot';
+import RobotoRegularWOFF from './fonts/Roboto-Regular.woff';
+import RobotoRegularTTF from './fonts/Roboto-Regular.ttf';
+
+import CustomSubHeader from 'components/CustomSubHeaders/CustomSubHeader';
+import CustomMobileSubHeader from 'components/CustomSubHeaders/CustomMobileSubHeader';
+
+const domainName = 'https://filecoin.io/';
+
+const colors = {
+  firm: '#0090FF',
+  elText: '#000000',
+  mainText: '#7A7A7A',
+  link: '#7A7AF0',
+  bg: '#FFFFFF',
+  extraBg: '#F3F3F3',
+  menuItemText: '#999999',
+  selectedMenuItemText: '#000000',
+};
+
+export const CustomSubHeaderConfig = {
+  design: 'filecoin_style',
+  styles: {
+    bg: {
+      header: colors.bg ?? '#FFFFFF',
```

Comment:
You have different style of colors: uppercase and lowercase `FFFFFF/ffffff`.
Also you can import some colors from `style-constants.js`. Also your unique colors can me moved to it's own file with all related colors